usersession/agent/rest_api.go

```diff
@@ -43,8 +46,18 @@ var restApi = []*Command{
 	sessionInfoCmd,
 	serviceControlCmd,
 	pendingRefreshNotificationCmd,
+	beginDeferredRefreshNotification,
+	finishDeferredRefreshNotification,
+	updateRefreshStatusNotification,
 }
 
+type NotifyRefreshOperation int
+
+const (
+	ShowNewNotification NotifyRefreshOperation = 0
+	DestroyNotification                        = 1
```

Comment:
Nitpick, this can be simplified
```suggestion
	ShowNewNotification NotifyRefreshOperation = iota
	DestroyNotification
```

Reply:
Good point!

Reply:
Done.